src/app/views/authentication/Authentication.tsx

```diff
@@ -0,0 +1,142 @@
+import hello from 'hellojs';
+import { PrimaryButton } from 'office-ui-fabric-react';
+import React, { Component } from 'react';
+
+import { connect } from 'react-redux';
+import { bindActionCreators, Dispatch } from 'redux';
+
+import { IAuthenticationProps, IAuthenticationState } from '../../../types/authentication';
+import * as authActionCreators from '../../services/actions/auth-action-creators';
+import * as queryActionCreators from '../../services/actions/query-action-creators';
+import { ADMIN_AUTH_URL, AUTH_URL, DEFAULT_USER_SCOPES , TOKEN_URL, USER_INFO_URL } from '../../services/constants';
+import './authentication.scss';
+
+export class Authentication extends Component<IAuthenticationProps,  IAuthenticationState> {
+  public state = {
```

Comment:
I prefer using a constructor because it aligns with the OOP model as compared to a class variable.